.github/workflows/run_cli_on_all_repos.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow only runs when manually triggered, and calls a reusable workflow to run the CLI on all repos in OpenNeuroDatasets-JSONLD.
+name: Run CLI on all repos
+
+on:
+  workflow_dispatch:
+
+jobs:
+    get-repos:
+      name: Get repos
+      runs-on: ubuntu-latest
+      outputs:
+        dataset_list: ${{ steps.get-repos.outputs.repos }}
+
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+
+        # This step gets a list of all repos in OpenNeuroDatasets-JSONLD, and fetches each repo's latest SHA
+        # Each repo ID and its corresponding SHA are written to a job output variable "repos"
+        #
+        # We use syntax here to ensure that "repos" is a multiline string, where each line represents one repo
+        # See: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings
```

Comment:
Same here, please move this down to line 40 ff to make sure we have it as reference when looking at this funky multiline syntax